kairon/shared/account/activity_log.py

```
@@ -56,6 +56,18 @@ def add_user_activity_log(a_type: UserActivityType, account: int = None,  email:
             raise AppException(str(e))
 
+    @staticmethod
+    def get_user_activity_log(email: str):
+        try:
+            user_activity_log = UserActivityLog.objects(
+                user=email, type=UserActivityType.user_consent.value
+            ).order_by("-timestamp").first()
+            user_activity_log = user_activity_log.to_mongo().to_dict()
+
+        except Exception as e:
+            raise AppException(str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)